client/src/shared/searchbar/searchbarController.js

```diff
@@ -9,6 +9,9 @@ sharedModule.controller('SearchbarController', ['$state', function SearchbarCont
             query: this.input,
         });
     };
+    this.clearSearchInput=function(){
+        this.input = null;
```

Comment:
Please change null to empty string. Changing variable types can slow down performance dramatically

Reply:
Updated. Thanks!